webapp/src/components/rhs/rhs_post_update.tsx

```diff
@@ -34,21 +36,33 @@ const RHSPostUpdate = (props: Props) => {
     const isNextUpdateScheduled = props.playbookRun.previous_reminder !== 0;
     const timestamp = getTimestamp(props.playbookRun, isNextUpdateScheduled);
     const isDue = isNextUpdateScheduled && timestamp.isBefore(now);
+    const isFinished = props.playbookRun.current_status === PlaybookRunStatus.Finished;
 
     let pretext = 'Last update';
-    if (isNextUpdateScheduled) {
+    if (isFinished) {
+        pretext = 'Run finished';
+    } else if (isNextUpdateScheduled) {
         pretext = (isDue ? 'Update overdue' : 'Update due');
     }
 
     const timespec = (isDue || !isNextUpdateScheduled) ? PastTimeSpec : FutureTimeSpec;
 
+    let icon;
+    if (isFinished) {
+        icon = <Icon path={mdiFlagCheckered}/>;
+    } else if (isDue) {
+        icon = Exclamation;
+    } else {
+        icon = Clock;
```

Comment:
The exclamation and clock icons are not visible now, we need to instantiate them. I don't know why Typescript is ok with the current code, by the way, shouldn't it complain?

```suggestion
        icon = <Exclamation/>;
    } else {
        icon = <Clock/>;
```

Reply:
Ooh, good catch: didn't even notice that! I think TypeScript doesn't complain since I didn't type the variable, and rendering the function is "ok" but probably results in nothing? Fixed with typing:

<img width="682" alt="CleanShot 2021-09-23 at 13 24 59@2x" src="https://user-images.githubusercontent.com/1023171/134546377-79730d71-c0fd-43fb-ba57-7cd1ae8871e3.png">

Reply:
Cool!